Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.yml

```diff
@@ -118,177 +127,344 @@ script:
       required: false
       secret: false
     deprecated: false
-    description: Retrieve alerts
+    description: Search HelloWorld Alerts.
     execution: false
     name: helloworld-search-alerts
     outputs:
-    - contextPath: HelloWorld.Alert.id
-      description: Alert id
-      type: string
+    - contextPath: HelloWorld.Alert.alert_id
+      description: Alert ID.
+      type: String
+    - contextPath: HelloWorld.Alert.alert_status
+      description: Alert status. Can be 'ACTIVE' or 'CLOSED'.
+      type: String
+    - contextPath: HelloWorld.Alert.alert_type
+      description: Alert type.
```

Comment:
we should document all the known enum types. alert_type, status, etc. If date field is returned then give an example in order to show the format of that date

Reply:
Alert type can be a lot of thing, doesn't have a predefined list of types. I put a note in there.